applet/libapplets/web_wifi/source/main.c

```diff
@@ -0,0 +1,63 @@
+// Include the most common headers from the C standard library
+#include <stdio.h>
+#include <stdlib.h>
+#include <string.h>
+#include <malloc.h>
+
+// Include the main libnx system header, for Switch development
+#include <switch.h>
+
+// See also libnx web.h.
+
+// This example shows how to use the WebWifiAuth LibraryApplet.
+
+// Main program entrypoint
+int main(int argc, char* argv[])
+{
+    Result rc=0;
+
+    // This example uses a text console, as a simple way to output text to the screen.
+    // If you want to write a software-rendered graphics application,
+    //   take a look at the graphics/simplegfx example, which uses the libnx gfx API instead.
+    // If on the other hand you want to write an OpenGL based application,
+    //   take a look at the graphics/opengl set of examples, which uses EGL instead.
+    consoleInit(NULL);
+
+    printf("webWifi example\n");
+
+    consoleUpdate(NULL);
+
+    WebWifiConfig config;
+
+    // Set the initial URL that the applet will navigate too.
+    webWifiCreate(&config, "http://example.org/");
+
+    printf("Running webWifiShow...\n");
+    rc = webWifiShow(&config);
+    printf("webWifiShow(): 0x%x\n", rc);
+
+    printf("Press + to exit.\n");
+
+    // Main loop
+    while (appletMainLoop())//This loop will automatically exit when applet requests exit.
+    {
+        // Scan all the inputs. This should be done once for each frame
+        hidScanInput();
+
+        // hidKeysDown returns information about which buttons have been
+        // just pressed in this frame compared to the previous one
+        u64 kDown = hidKeysDown(CONTROLLER_P1_AUTO);
+
+        if (kDown & KEY_PLUS)
+            break; // break in order to return to hbmenu
+
+        // Your code goes here
```

Comment:
Might want to perform the web applet launch when a button is pressed instead of doing it immediately on startup.